CRM/Core/DAO/AllCoreTables.php

```diff
@@ -166,7 +171,8 @@ public static function multilingualize($class, $originalIndices) {
    *   [EntityName => CRM_DAO_ClassName]
    */
   public static function daoToClass() {
-    return array_combine(array_keys(self::getEntities()), array_column(self::getEntities(), 'class'));
+    $entities = self::getEntities();
+    return array_combine(array_keys($entities), array_column($entities, 'class'));
```

Comment:
This is just for performance. Because `self::getEntities` is no longer a do-nothing function, let's not call it twice for no reason.